docs/reference/TensorFlow.md

```diff
@@ -1,405 +1,394 @@
+This is especially exciting for IoT, because lower network use means lower power consumption. You can also better guarantee the security and privacy of your users, since you do not need to send data back to the cloud unless you know for sure that it is relevant.
 
-This is especially exciting for IoT, because less network utilization means lower power consumption. Also, you can better guarantee the security and privacy of your users, since you do not need to send data back to the cloud unless you know for sure that it is relevant.
+In this guide, you will learn how to perform machine learning inference on an Arm Cortex-M microcontroller with TensorFlow Lite for Microcontrollers. You will deploy a sample application we wrote that uses the microphone on the K66F and a TensorFlow machine learning model to detect the words “yes” and “no”.
 
-In the following guide, you will learn how you can perform machine learning inference on an Arm Cortex-M microcontroller with TensorFlow Lite for Microcontrollers.
+The guide has step by step instructions for installing and running the application, followed by information about how TensorFlow and the application work. Finally, it shows you how to retrain the ML model with new words.
 
-**About TensorFlow Lite**
+## Using the example application
 
-TensorFlow Lite is a set of tools for running machine learning models on-device. TensorFlow Lite powers billions of mobile app installs, including Google Photos, Gmail, and devices made by Nest and Google Home.
-
-With the launch of TensorFlow Lite for Microcontrollers, developers can run machine learning inference on extremely low-powered devices, like the Cortex-M microcontroller series. Watch the video to learn more about the announcement:
-
-[![An introduction to TensorFlow Lite](http://img.youtube.com/vi/DKosV_-4pdQ/0.jpg)](http://www.youtube.com/watch?v=DKosV_-4pdQ)
-
-# Getting Started
+<!--this needs to move-->
+**Important:** We recommend running the following commands from inside the Mbed CLI terminal that gets launched with the Mbed CLI Application. This is because it will be much quicker to set up, because it resolves all your environment dependencies automatically.
 
-**Before you begin**
+### Prerequisites
 
 Here is what you will need to complete the guide:
 
-* Computer that supports Mbed CLI (version 1.10.4)
-
-* NXP FRDM K66F
-
-* Mini-USB cable
-
-* Python 2.7. Using pyenv is recommended to manage python versions.
+* Ubuntu 20.04 LTS. If you are using a Windows machine, install [VirtualBox](https://www.virtualbox.org/wiki/Downloads)<!--originally you said 6.0 - that version's support ended in July. Can they install the latest version, or do they need 6.0 specifically?--> and then [Install Ubunto on it](https://www.virtualbox.org/wiki/Linux_Downloads).
```

Comment:
Regarding VirtualBox, it should not make a difference as long as they are able to run Ubuntu 20.04 on it. Although their website states "Please also use version 6.0 if you need to run VMs with software virtualization, as this has been discontinued in 6.1. Version 6.0 will remain supported until July 2020". So, I would suggest that for the sake of consistency users stick with version 6.0 to ensure the demo works accordingly. 

 Also, there is a tiny spelling mistake, it should be "[Install Ubuntu on it]", not "[Install Ubunto on it]".

Reply:
```suggestion
* Ubuntu 20.04 LTS. If you are using a Windows machine, install [VirtualBox](https://www.virtualbox.org/wiki/Downloads) and then [Install Ubuntu on it](https://www.virtualbox.org/wiki/Linux_Downloads).
```

Reply:
Agreed can we say: 

> This tutorial has been tested on Ubuntu 20.04 LTS. If you are using a Windows machine, we recommend using virtualization software to create a virtual Linux machine to run this tutorial.